### PR TITLE
Get a easy way to start AP with the bootstrap switch

### PR DESCRIPTION
**Pull request by LeSuedois**

### CODE REVIEW

File: src/CoolBoard.cpp [hunk 1]
```diff
@@ -1310,4 +1314,27 @@ bool CoolBoard::sendPublicIP()
...
+}
+
+/**
+ *  CoolBoard::startAP():
+ *  This method is provided to check if the user 
+ *  used the run/load switch to start the AP
+ *  for further configuration/download
+ *
+ */
+void CoolBoard::startAP()
```
Comment (3rwww1) [inline src/CoolBoard.cpp]: formatting (brace on same lien as `if` statement)

File: src/CoolBoard.cpp [hunk 2]
```diff
@@ -1310,4 +1314,27 @@ bool CoolBoard::sendPublicIP()
...
+{
+#if DEBUG == 1
+  Serial.println(F("Entering Coolboard.startAP"));
+#endif
+  Serial.print(F("Bootstrap Switch : "));
+  Serial.println(digitalRead(Bootstrap));
+  if (digitalRead(Bootstrap) == LOW)
```
Comment (3rwww1) [inline src/CoolBoard.cpp]: formatting (brace on same lien as `if` statement)

File: src/CoolBoard.h [hunk 3]
```diff
@@ -214,6 +216,13 @@ class CoolBoard {
    *register latch All I2C is over pins (2,14)
    */
   const int EnI2C = 5;
+
+  /**
+   *  Bootstrap pin,
+   *  double usage for flashing the Coolboard 
+   *  and start up the AP for further configuration
+   */
+  const int Bootstrap = 0;
```
Comment (3rwww1) [inline src/CoolBoard.h]: variables should be written in `camelCase` and not `PascalCase`, e.g. `bootstrap` and `enI2C` instead of `Bootstrap` and `EnI2C`

**Review by 3rwww1 — CHANGES_REQUESTED** [review]
Some very minor changes to make, otherwise, looks great!

*However this brings the question of how to expose load and reset button when the CoolBoard is in inside a casing*

File: src/CoolBoard.h (hunk 3 above)
Comment (LeSuedois) [reply src/CoolBoard.h]: There are some more of these PascalCase in the lib, take a look at CoolBoardSensors.h..
by the way, isn't CoolBoardSensors PascalCase too? 
maybe we should make a separate commit to clear the camel/PascalCase issue. at least I will create the issue to keep a trace of it...

Comment (3rwww1) [reply src/CoolBoard.h]: Yes, but `CoolBoardSensors` is a **class** name and as per conventions, `ClassNames` are `PascalCase`

Of course we should make separate commits to fix this kind of stuff in other places, but keep new code clear of those tiny compliance issues and rename the new `Bootstrap` variable to `bootstrap`

**Pull request merged**